packaging/pack.proj

```diff
@@ -0,0 +1,30 @@
+<Project ToolsVersion="15.0">   
+    <Import Project="..\build\common.props" />
+
+    <Target Name="Pack"> 
+        <MSBuild Projects="$(MSBuildThisFileDirectory)..\src\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.csproj"
```

Comment:
Could you declare `<SolutionRoot>$(GetDirectoryNameOfFileAbove(MvcPrecompilation.sln)/</SolutionRoot>` and use that instead of doing `$(MSBuildThisFileDirectory)..\` everywhere in this file?